src/test/codegen/unwind-abis/c-unwind-abi-panic-abort.rs

```diff
@@ -9,7 +9,8 @@
 // CHECK: @rust_item_that_can_unwind() unnamed_addr [[ATTR0:#[0-9]+]]
 #[no_mangle]
 pub unsafe extern "C-unwind" fn rust_item_that_can_unwind() {
-    // CHECK: call void @_ZN4core9panicking15panic_no_unwind
+    // Handle both legacy and v0 symbol mangling.
+    // CHECK: call void @{{_ZN4core9panicking15panic_no_unwind|_RNvNtCsRkAe6OifLQ_4core9panicking15panic_no_unwind}}
```

Comment:
Matching `_RNvNtCsRkAe6OifLQ_` will break every time the rustc version changes. To be precise `RkAe6OifLQ` is the encoding of the stable crate id, which is not stable across rustc versions, only across different compilations of a crate with the same rustc version and `-Cmetadata` arguments.

Reply:
Thank you! Replaced these.